src/libraries/Microsoft.PowerFx.Core/Syntax/UserDefinitions.cs

```diff
@@ -106,12 +89,14 @@ private IEnumerable<UserDefinedFunction> CreateUserDefinedFunctions(IEnumerable<
                     continue;
                 }
 
-                if (!(CheckParameters(udf.Args, errors) & CheckReturnType(udf.ReturnType, errors)))
```

Comment:
>&  [](http://example.com/codeflow?start=56&length=2)

For context:
[https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators)